src/tools/debugger/session/Breakpoints.java

```diff
@@ -100,11 +109,8 @@ public synchronized void addOrUpdateBeforeExpression(final SectionBreakpoint bId
 
   public synchronized void addOrUpdateAfterExpression(final SectionBreakpoint bId) {
     // TODO: does this work???
```

Comment:
what happens here? I saw that is used for transactions and fork/join but don't we have tests for them?